test/libsolidity/smtCheckerTests/functions/functions_identifier_nested_tuple_2.sol

```diff
@@ -0,0 +1,21 @@
+pragma experimental SMTChecker;
+
+library L {
+	struct S {
+		uint256[] data;
+	}
+	function f(S memory _s) internal {
+	}
+}
+
+contract C {
+	using L for L.S;
+	function f() public returns (uint256) {
+		L.S memory x;
+		(x.f)();
```

Comment:
Maybe add a return value to `f` and use it in an assertion here?

Reply:
Updated.